mix.exs

```diff
@@ -21,6 +21,8 @@ defmodule TicTacToeElixir.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
+      {:ecto_sql, "~> 3.0"},
```

Comment:
**thought:** If you consider your walking skeleton, you might want to apply a sort of slimmed-down version of that when you add in something architecturally significant to your application - this should give you the confidence that you can spin up a database in CI, apply your migrations (rarely a problem with a single migration, mind) and run a basic test (what might that look like in terms of a database?)